zarr-in-practice.ipynb

```diff
@@ -0,0 +1,3003 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    # Here we create a simple Zar store.
</pre><p>Typo: <code>Zar</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/cloud-optimized-geospatial-formats-guide/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>